README.md

```diff
@@ -6,7 +6,7 @@
 
 Membrane H264 parser.
 It is the Membrane element responsible for parsing the incoming h264 stream. The parsing is done as a sequence of the following steps:
-* splitting the h264 stream into stream NAL units, based on the "Annex B" of the "ITU-T Rec. H.264 (01/2012)"
+* splitting the h264 stream into stream NAL units, based on the "ITU-T Rec. H.264 (01/2012)"
```

Comment:
```suggestion
* splitting the H264 stream into stream NAL units, based on the "ITU-T Rec. H.264 (01/2012)"
```
We can stick to that formatting ;)

Reply:
I believe that the splitting might be also based on length-prefix, which is not defined in ITU-T Rec. H.264 (01/2012)